src/auth/auth-domain.ts

```diff
@@ -20,6 +20,9 @@ const thirdPartyDomainSTSUrlMappings: { [domain: string]: string } = {
     'https://account.smartaidforparents-com-stage.smarttuition.net',
   'smartaidforparents.com': 'https://account.smartaidforparents.com',
   'smarttuition.com': 'https://account.smarttuition.com',
+  'yourcausegrants.com': 'https://www.yourcausegrants.com',
```

Comment:
Should these URLs be `https://sts.yourcausegrants.com` etc?

Reply:
I have updated these to have the proper URLs